WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -48,5 +55,38 @@ private static void ModifyRegistry(bool runOnSystemStartup, string pathToExeFile
 				key.DeleteValue(nameof(WalletWasabi), false);
 			}
 		}
+
+		private static void StartOnMacStartUp(bool runOnSystemStartup)
+		{
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+			{
+				throw new InvalidOperationException("Running osascript can only be done on macOS.");
+			}
+
+			ProcessStartInfo processInfo = new()
+			{
+				UseShellExecute = true,
+				WindowStyle = ProcessWindowStyle.Hidden,
+				FileName = "/bin/bash",
```

Comment:
There is already `EnvironmentHelpers.ShellExecAsync(...)` and it would be great to actually refactor that method and use that if it is possible. (it would verify that `/usr/bin/env sh` can/cannot be used)

Reply:
I will try it out. If it works, i will make the changes.

Reply:
It works perfectly. I really should look over the `Helpers`, bc i always re-write code pieces...